users/migrations/0013_replace_has_accepted_marketing_with_communication.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.11 on 2024-05-24 11:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    The `has_accepted_marketing` field was named badly,
+    since there is "no marketing messaging of any kind" in Kukkuu.
+    However, there is a need for a field that tells whether or not
+    a user allows messaging, like email sending,
+    related to event invitations, etc. Also, the default value of
+    that field should be `True` instead of `False`, since the current
+    users have "all accepted the messaging by creating a new account
+    to the Kukkuu" and they have not been able make any other decisions
+    by themselves (before now). The `has_accepted_marketing` has not
+    yet been in use for long and there are no decisions that could not
+    be overridden now when the Kukkuu is changing the auth service
+    from Tunnistamo to Keycloak (Helsinki-profile).
```

Comment:
The last sentence in this paragraph is a bit unclear to me, particularly this part "there are no decisions that could not    be overridden". What do you mean by this?